cloud_experts_tutorials/rosa-mobb-azure-group-claims.adoc

```diff
@@ -0,0 +1,149 @@
+
+[source,terminal]
+----
+$ rosa create idp \
+    --cluster ${CLUSTER_NAME} \ <1>
+    --type openid \
+    --name ${IDP_NAME} \ <2> 
+    --client-id ${APP_ID} \ <3> 
+    --client-secret ${CLIENT_SECRET} \ <4> 
+    --issuer-url https://login.microsoftonline.com/${TENANT_ID}/v2.0 \ <5>
+    --email-claims email \
+    --name-claims name \
+    --username-claims preferred_username \
+    --groups-claims groups \
+    --extra-scopes email,profile
+----
+--
+<1> Provide the name of your ROSA cluster with the `export CLUSTER_NAME=example-cluster` command in your terminal.
+<2> Provide the name you used in the OAuth callback URL with the `export IDP_NAME=AAD` command in your terminal.
+<3> Provide the Application (client) ID with the `export APP_ID=yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy` command in your terminal.
+<4> Provide the client secret with the `export CLIENT_SECRET=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` command in your terminal.
+<5> Provide the Directory (tenant) ID with the `export TENANT_ID=zzzzzzzz-zzzz-zzzz-zzzz-zzzzzzzzzzzz` command in your terminal.
+--
+
+[id="rosa-mobb-azure-group-claims-additional-permissions"]
+== Grant additional permissions to individual groups
+
+Once the cluster authentication Operator reconciles your changes (generally within a few minutes), you will be able to log in to the cluster using Azure AD. In addition, the cluster OAuth provider will automatically create or update the membership of groups the user is a member of (using the group ID).
+
+Once you log in, you will notice that you have very limited permissions. This is because, by default, OpenShift only grants you the ability to create new projects (namespaces) in the cluster. Other projects (namespaces) are restricted from view. The cluster OAth provider **does not** automatically create RoleBindings and ClusterRoleBindings for the groups that are created, you are responsible for creating those via your own processes.
+
+OpenShift includes a significant number of pre-configured roles, including the `cluster-admin` role that grants full access and control over the cluster. To grant an automatically generated group access to the `cluster-admin` role, you must create a ClusterRoleBinding to the group ID.
+
+[source,terminal]
+----
+$ oc create clusterrolebinding cluster-admin-group \
+    --clusterrole=cluster-admin \
+    --group=$GROUP_ID \ <1>
+----
+--
+<1> Provide your Azure Group ID that you would like to have cluster admin permissions with ` export GROUP_ID=wwwwwwww-wwww-wwww-wwww-wwwwwwwwwwww`.
```

Comment:
replace $GROUP_ID with <GROUP_ID> and change the highlight to:

Provide your Entra ID group ID that you would like to have cluster admin permissions.